source/scripts/Background/controllers/EVMChainController/etherscanApi.ts

```diff
@@ -59,17 +66,31 @@ export const getETHTransactionHistory = async ({
   endblock,
   apiKey,
 }: TransactionHistoryParam & { baseUrl: string; apiKey?: string }): Promise<Txs> => {
-  let url = baseUrl + `/api?module=account&action=txlist&sort=desc` + getApiKeyQueryParameter(apiKey)
-  if (address) url += `&address=${address}`
-  if (offset) url += `&offset=${offset}`
-  if (page) url += `&page=${page}`
-  if (startblock) url += `&startblock=${startblock}`
-  if (endblock) url += `&endblock=${endblock}`
+  let url =
+    baseUrl +
+    `/api?module=account&action=txlist&sort=desc` +
+    getApiKeyQueryParameter(apiKey);
+  if (address) url += `&address=${address}`;
+  if (offset) url += `&offset=${offset}`;
+  if (page) url += `&page=${page}`;
+  if (startblock) url += `&startblock=${startblock}`;
+  if (endblock) url += `&endblock=${endblock}`;
```

Comment:
Parameter chains should be done using the [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) class, besides that one, I would create a util fn to add parameters if they exist in an object (reduces lines & avoids using inline if statements).